### PR TITLE
Improve custom chain support

### PR DESCRIPTION
**Pull request by brunobar79**
Fixes BX-####
Figma link (if any):

## What changed (plus any additional context for devs)
- We now have asset prices, charts for all the chains that we unofficially support through the backend.
- We hide prices for chains that we don't have any info
- We have asset & chain icons for a lot of popular chains taken from the assets repo.

## Screen recordings / screenshots

http://recordit.co/5rM9BOLzol

### CODE REVIEW

File: src/core/resources/assets/customNetworkAssets.ts [hunk 1]
```diff
@@ -110,42 +140,85 @@ async function customNetworkAssetsFunction({
...
+              asset: customNetworkNativeAssetParsed,
+              currency,
+              balance: nativeAssetBalance.toString(),
+            });
 
-            parsedAssetsDict[chain.id as ChainId] = {
-              [customNetworkNativeAssetParsed.uniqueId]:
-                customNetworkNativeAssetParsed,
-            };
+            console.log(
```
Comment (estebanmino) [inline src/core/resources/assets/customNetworkAssets.ts:204]: log

File: src/entries/popup/components/TransactionFee/TransactionFee.tsx [hunk 2]
```diff
@@ -168,6 +201,12 @@ function Fee({
                   <TextOverflow weight="semibold" color="label" size="14pt">
                     {isLoading
                       ? '~'
+                      : !nativeCurrencyGasFeeSupported(chainId)
+                      ? convertToNativeGasFee(
```
Comment (estebanmino) [inline src/entries/popup/components/TransactionFee/TransactionFee.tsx:205]: if the intention of this was using the fee as native value if the price is not available that should have been done here https://github.com/rainbow-me/browser-extension/pull/1111

that should be checking if native asset has price and if not default to crypto values

Comment (estebanmino) [reply src/entries/popup/components/TransactionFee/TransactionFee.tsx:205]: to fix it you can change https://github.com/rainbow-me/browser-extension/blob/master/src/entries/popup/hooks/useGas.ts#L54C23-L54C47

`useNativeAssetForNetwork` to `useNativeAsset`

**Review by estebanmino — APPROVED** [review]

**Review by derHowie — APPROVED** [review]
lgtm

**Pull request merged**